README.md

```diff
@@ -1,8 +1,6 @@
 # Terraform Harbor Provider
 [![GitHub release](https://img.shields.io/github/release/goharbor/terraform-provider-harbor.svg)](https://github.com/goharbor/terraform-provider-harbor/releases/)
 [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fgoharbor%2Fterraform-provider-harbor.svg?type=shield)](https://app.fossa.com/projects/git%2Bgithub.com%2Fgoharbor%2Fterraform-provider-harbor?ref=badge_shield)
-#[![CircleCI](https://circleci.com/gh/goharbor/terraform-provider-harbor.svg?style=svg)](https://circleci.com/gh/goharbor/terraform-provider-harbor)
-#[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=goharbor_terraform-provider-harbor&metric=alert_status)](https://sonarcloud.io/dashboard?id=goharbor_terraform-provider-harbor)
```

Comment:
You can now change for this one here 

[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=terraform-provider-harbor&metric=alert_status)](https://sonarcloud.io/summary/new_code?id=terraform-provider-harbor)

Reply:
okay, done
anything we can do about the "quality gate not computed" ?